deploy/scripts/src/mobile/test.ts

```diff
@@ -65,6 +55,12 @@ export function setup (): void {
   describeProfile(loadProfile)
 }
 
+export function dcmawStartAuthorizeRequest (): void {
+  const jar = http.cookieJar()
+  const sessionId = getSessionId()
+  setSessionCookie(jar, sessionId)
```

Comment:
Do we need to set the session cookie here?

Reply:
Actually, no, both calls are made inside getSessionId().

Reply:
So actually, if we remove the setSessionCookie we'll have es lint errors "'sessionId' is assigned a value but never used.eslint[@typescript-eslint/no-unused-vars]"
Job: Run linting and unit tests -> will fail